tests/integration/test_web_client.py

```diff
@@ -462,3 +462,17 @@ def test_creation_lifecycle(self, antares_web: AntaresWebDesktop):
         empty_settings = StudySettings()
         new_study.update_settings(empty_settings)
         assert old_settings == new_study.get_settings()
+
+    def test_create_variant(self, antares_web: AntaresWebDesktop):
```

Comment:
Using the fixture antares_web launches the Desktop version which can take a long time. I would prefer you put this code inside the other test. Otherwise the test is nice (you could add the same test on the other creation method you introduced)